src/NServiceBus.Testing/Saga.cs

```diff
@@ -345,7 +345,7 @@ public Saga<T> AssertSagaCompletionIs(bool complete)
         /// </summary>
         public Saga<T> ExpectTimeoutToBeSetIn<TMessage>(Func<TMessage, TimeSpan, bool> check = null)
         {
-            testableMessageHandlerContext.ExpectedInvocations.Add(new ExpectDelayDeliveryWith<TMessage>(check));
+            testContext.ExpectedInvocations.Add(new ExpectDelayDeliveryWith<TMessage>(check));
```

Comment:
While looking at the usage of `ExpectedInvocations` I started wondering if the additional property actually has benefits? What if we'd just do something like

```
testContext.Add<TExpectation>(TExpectation expectation) where TExpectation : WhatEverBaseClassOrInterfaceWeHaveHere
```
?